src/vos/lru_array.h

```diff
@@ -497,4 +499,12 @@ lrua_array_free(struct lru_array *array);
 void
 lrua_array_aggregate(struct lru_array *array);
 
+static inline void
+lrua_refresh_key(struct lru_entry *entry, uint64_t key)
+{
+	D_ASSERT(entry != NULL);
+
+	entry->le_key = key;
```

Comment:
The entry is found by the address instead of the old key, so it must be the original entry. So the sanity check of the old key is unnecessary. Otherwise, if the entry could be evicted by race, then the address may become invalid as to segment fault when access it.